### PR TITLE
Adding file writer (used by the logger)

### PR DESCRIPTION
**Pull request by terah**

### CODE REVIEW

File: src/Util/Writer/File.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+     * @param  string $content
+     */
+    public function write($content)
+    {
+        fwrite($this->resource, $content);
+    }
+
+    public function _destruct()
+    {
+        fclose($this->resource);
```
Comment (duncan3dc) [inline src/Util/Writer/File.php]: Should we close resources we didn't open?

**Comment by terah** [comment]
I thought about that... Then I thought about fopening the resource in the constructor but I think that's a bit limiting.  It's a tricky one.  Probably should go in the calling class' destructor.  

On another note, I think the fwrite can be changed to a gzwrite as I think it's backwards compatible with uncompressed resources.  If that's true, we can optionally create gzipped logs.

**Comment by terah** [comment]
Here's another go: https://github.com/terah/climate/blob/master/src/Util/Writer/File.php

Let me know your thoughts...

**Comment by joetannenbaum** [comment]
Thanks @terah!

**Pull request merged**